android/src/main/java/com/reactnativecommunity/netinfo/NetworkCallbackConnectivityReceiver.java

```diff
@@ -56,10 +56,12 @@ void unregister() {
     }
 
     @SuppressLint("MissingPermission")
-    private void updateAndSend() {
+        void updateAndSend() {
```

Comment:
Maybe this is just a Github display issue, but the intending look wrong here. `void` should align with `@SuppressLint`.

Reply:
Updated